### DISCUSSION TITLE
Speed Limiter

### DISCUSSION THREAD

**Discussion by cbrueg** [DIGITALCRIMINALS/OnlyFans#393] [Ideas]
Hey All,

is there any (Easy) way to restrict the tools max Dowload speed ?

or did you plan to implement something like this ?

**Comment by Nick05491** [comment]
Out of curiosity, why do you want to restrict download speed?

I'm having a lot of issues in the last month or so with files not being downloaded.  A single profile usually downloads fine, but when I bulk scrape all subscriptions barely any files are downloaded.  I've considered whether the OF server is rejecting the requests because of the rate at which the downloader is requesting them.  A pause or way of limiting them number of requests may fix this.  Just wondering if you're having similar issues.

**Reply by cbrueg** [reply]
I have  had  same Problems , but the reason why i want to Limit the Speed is that I tried some Automataition which Resulted in a complete loss of all data :(  and I had the Problem that I had to re Download mor then 500GB of Files witch blocked my entire Internet for more than a day  and i had to pause the Script to be able to Work from home :)
